cpmpy/solvers/ortools.py

```diff
@@ -97,17 +86,17 @@ def __add__(self, cons):
         """
         # store new user vars
         new_user_vars = get_variables(cons)
-        for v in frozenset(new_user_vars)-frozenset(self.user_vars):
-            self.user_vars.append(v)
+        for v in frozenset(new_user_vars) - frozenset(self.user_vars):
+            self.user_vars.add(v)
```

Comment:
now that user_vars is a set you can do user_vars.update(get_vars(cons)) or simlar, as you do in the pysat interface?